packages/core-mobile/app/services/earn/exportC.ts

```diff
@@ -53,15 +52,13 @@ export async function exportC({
 
   const cChainBalanceAvax = AvaxC.fromWei(cChainBalanceWei)
   const requiredAmountAvax = AvaxC.fromWei(requiredAmountWei)
-  const pChainFeeAvax = await calculatePChainFee(avaxXPNetwork)
-  const amountAvax = requiredAmountAvax.add(pChainFeeAvax)
 
```

Comment:
pointed out by @neven-s. we don't need to include the p static fee here since we already include it in the required amount.